src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -731,14 +732,14 @@ def test_process_unsupported_declarations(fortran_reader):
     # with unknown type.
     reader = FortranStringReader("real:: a = 1.1")
     fparser2spec = Specification_Part(reader).content[0]
-    processor.process_declarations(fake_parent, [fparser2spec], [])
+    processor.process_declarations(fake_parent, [fparser2spec], [], {})
```

Comment:
You have added a `{}` argument in all the `process_declarations` from here to the end of this file? Is this on purpose or is it the remains of an intermediate change, as it seems unnecessary because it is already the optional default value.

Reply:
Argh. It is the remains of an intermediate change. I thought I'd gone through and fixed all of them, sorry.